### PR TITLE
Update Japanese Translation

### PR DESCRIPTION
**Pull request by tkusano**
- Unify the translation of "video" as "ビデオ", not "動画".
- Follow datetime and onboarding related updates.

### CODE REVIEW

**Comment by tkusano** [comment]
Could you please review the translation?
@dolciss  @Hima-Zinn  @middlingphys  @hibiki

**Review by Hima-Zinn — COMMENTED** [review]
It looks fine.

**Comment by tkusano** [comment]
Translated 'video caption/subtitles' and 'QR of starterpacks' related messages.

File: src/locale/locales/ja/messages.po [hunk 1]
```diff
@@ -4003,6 +4019,10 @@ msgstr "{0}のみ返信可能。"
 msgid "Only contains letters, numbers, and hyphens"
 msgstr "英数字とハイフンのみ"
 
+#: src/view/com/composer/videos/SubtitleFilePicker.tsx:31
+msgid "Only WebVTT (.vtt) files are supported"
+msgstr "WebVTT（.vtt）ファイルのみに対応してます"
```
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:4024]:
```suggestion
msgstr "WebVTT（.vtt）ファイルのみに対応しています"
```

Comment (tkusano) [reply src/locale/locales/ja/messages.po]: 修正しました!

**Comment by tkusano** [comment]
Translated video processing error messages

**Comment by tkusano** [comment]
follow https://github.com/bluesky-social/social-app/commit/515f87ed2487d6d875dfc6a266e47e7785e94818 and https://github.com/bluesky-social/social-app/commit/4d97a2aa16083887a9b346d82e3b865e8a000355

**Comment by tkusano** [comment]
https://github.com/bluesky-social/social-app/commit/117926357d3a59db8fb12f9486f657c7b0f1cf69 related updates

**Comment by tkusano** [comment]
Translate video related messages.

**Review by pfrazee — APPROVED** [review]
Thanks all!

**Pull request merged**